### DISCUSSION TITLE
Have you found the discussion section here responsive or helpful?

### DISCUSSION THREAD

**Discussion by 3point14159** [horizontalsystems/unstoppable-wallet-android#5407] [Polls]
If so, please share how you actually engaged the development team. This seems to be the best non-functional wallet out there. Still can't use BIP39 passwords with special characters, despite the bug being fixed for over a month now. Why bother making a wallet if you cannot release a functional version in a timely manner? Coinomi abandonware actually works better than this wallet, and they don't even have any active development.

**Comment by esengulov** [comment]
> Discussion options
> ##
> [ ](/3point14159)

If you're referring to ^ special charter that should be available in the next version which is due to be releast early next week. To our knowledge support for all other special chartaers should be already in place.

**Reply by 3point14159** [reply]
I reported the lack of full-character support over a month ago and has been [fixed for some time now](https://github.com/horizontalsystems/unstoppable-wallet-android/issues/5290), just no maintenance release for over a month. You guys keep shifting the release back. [Earlier it was by the end of this week](https://www.reddit.com/r/UNSTOPPABLEWallet/comments/xoehel/comment/ipysumo/?utm_source=reddit&utm_medium=web2x&context=3), now it is next week. No one is holding their breath. You guys need some better coders, this is a mess.

**Reply by esengulov** [reply]
> I reported the lack of full-character support over a month ago and has been [fixed for some time now](https://github.com/horizontalsystems/unstoppable-wallet-android/issues/5290), just no maintenance release for over a month. You guys keep shifting the release back. [Earlier it was by the end of this week](https://www.reddit.com/r/UNSTOPPABLEWallet/comments/xoehel/comment/ipysumo/?utm_source=reddit&utm_medium=web2x&context=3), now it is next week. No one is holding their breath. You guys need some better coders, this is a mess.

Planning development sprints is not an easy task. When it comes to our coding know how our work speaks for itself. We were the first om implement Bitcoin lite client implementation on both Swift and Kotlin.

**Reply by esengulov** [reply]
That said, you do have a valid point regarding delayed release dates. I wish we could be better at planning but keep in mind we are fully self funded and are not getting paid by anyone for the work we do here.

**Reply by 3point14159** [reply]
I do keep in mind the self-funding/unpaid labor. That's all the more reason why there should be a better release strategy for bug fixes, while keeping new features and non-essential developments further down the release cycle. There is no reason to not have a bi-weekly bug fix release, then save the NFT, and whatever the newest hype is, for future releases, instead of doing all at the same time. By ensuring you have a working wallet, that can allow for more people (like me) to even be able to throw/donate some crypto your way.

**Reply by esengulov** [reply]
We will act on your feedback!

**Reply by 3point14159** [reply]
Look forward to seeing the improvements. I don't see anywhere in the app where to send donations, may want to include that in the about section of the app for users to directly send donations. If there is a link with donation addresses, please post it here.
